Personal_Portfolio.css

```diff
@@ -91,16 +100,19 @@ height: 1040px;
 background: #2d343c;
 
 }
+
 h2{
 font-size: "50px";
 font-family: verdana;
 line-height: 50px;
 }
+
 @media only screen and (max-width: 600px) {
   body {
-    background-color: lightblue;
+    background-color: white;
   }
 }
```

Comment:
🔳 Please consider pushing this styling to the top of the stylesheet, it is general styling that belongs to the total body elements, and this will make your stylesheet more organized.